Tasks/Gulp/typings/vsts-task-lib.d.ts

```diff
@@ -0,0 +1,564 @@
+declare module 'vsts-task-lib/taskcommand' {
```

Comment:
shouldn't this file be delivered with the lib, or is this required because it's an older version of the lib that doesn't deliver a d.ts file?